website/content/en/preview/getting-started/cloudformation.yaml

```diff
@@ -57,4 +53,20 @@ Resources:
               - ec2:DescribeInstanceTypes
               - ec2:DescribeInstanceTypeOfferings
               - ec2:DescribeAvailabilityZones
+          - Effect: Allow
+            Resource: "*"
+            Action:
+              - ec2:RunInstances
+              - ec2:TerminateInstances
+              - ec2:DeleteLaunchTemplate
+            Condition:
+              StringEquals:
+                "ec2:ResourceTag/karpenter.sh/discovery": !Ref ClusterName
```

Comment:
Just a note that we'll need to put this into release notes for resources provisioned before we started tagging things with `karpenter.sh/discovery` which was fairly recently I believe.

Reply:
is that something I should add somewhere or just note to self 😅

Reply:
I believe the Karpenter provisioner spec in our getting started guide will have to be updated to account for this condition.  Curious if you find the same when testing out the changes.

```
apiVersion: karpenter.sh/v1alpha5
kind: Provisioner
metadata:
  name: default
spec:
  tags:
    karpenter.sh/discovery: ${CLUSTER_NAME}
```

Reply:
added